### PR TITLE
OHRM5X-1331: Fix Language strings marked as not fixable

### PR DESCRIPTION
**Pull request by ManulMax**

### CODE REVIEW

File: src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue [hunk 1]
```diff
@@ -52,15 +52,34 @@ import {navigate} from '@ohrm/core/util/helper/navigation';
 import {APIService} from '@ohrm/core/util/services/api.service';
 import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
 import Spinner from '@ohrm/oxd/core/components/Loader/Spinner';
+import usei18n from '@/core/util/composable/usei18n';
+
+const {$t} = usei18n();
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue:57]: *

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue (hunk 1 above)
Comment (Super-Chama) [inline src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue:57]: Discussed before. please use the composables inside setup context

File: src/client/src/orangehrmAdminPlugin/pages/systemUser/SystemUser.vue [hunk 2]
```diff
@@ -150,14 +151,22 @@ export default {
...
+          _displayName = `${$t('general.ess')}`;
+        }
+        if (_displayName === 'Admin') {
+          _displayName = `${$t('general.admin')}`;
+        }
         return {
           id: item.id,
           userName: item.userName,
-          role: item.userRole?.displayName,
+          role: _displayName,
```
Comment (Super-Chama) [inline src/client/src/orangehrmAdminPlugin/pages/systemUser/SystemUser.vue:169]: `role: item.userRole?.displayName === 'Admin' ? $t('general.admin') : $t('general.ess')`

File: src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue [hunk 3]
```diff
@@ -52,15 +52,34 @@ import {navigate} from '@ohrm/core/util/helper/navigation';
...
+      _type = `${$t('admin.leave_approvals')}`;
+    }
+    if (_type === 'Leave Assignments') {
+      _type = `${$t('admin.leave_assignments')}`;
+    }
+    if (_type === 'Leave Cancellations') {
+      _type = `${$t('admin.leave_cancellation')}`;
+    }
+    if (_type === 'Leave Rejections') {
+      _type = `${$t('admin.leave_rejections')}`;
```
Comment (Super-Chama) [inline src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue:78]: redundant use of back-ticks and template literals. simply use as
`_type = $t('admin.leave_rejections')`

**Review by Super-Chama — CHANGES_REQUESTED** [review]

File: src/client/src/orangehrmTimePlugin/pages/timesheets/ViewEmployeeTimesheet.vue [hunk 4]
```diff
@@ -41,8 +41,33 @@
         ></timesheet-period>
       </template>
       <template #footer-title>
-        <oxd-text v-show="timesheetStatus" type="subtitle-2">
-          {{ $t('general.status') }}: {{ timesheetStatus }}
+        <oxd-text
+          v-show="timesheetStatus"
```
Comment (Super-Chama) [inline src/client/src/orangehrmTimePlugin/pages/timesheets/ViewEmployeeTimesheet.vue:45]: Please refactor this to a switch statement/ computed property. don't duplicate the template unnecessarily

File: src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue [hunk 5]
```diff
@@ -52,15 +52,35 @@ import {navigate} from '@ohrm/core/util/helper/navigation';
 import {APIService} from '@ohrm/core/util/services/api.service';
 import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
 import Spinner from '@ohrm/oxd/core/components/Loader/Spinner';
+import usei18n from '@/core/util/composable/usei18n';
 
 const subscribersNormalizer = data => {
+  const {$t} = usei18n();
```
Comment (Super-Chama) [inline src/client/src/orangehrmAdminPlugin/pages/emailSubscription/EmailSubscription.vue:58]: move this to setup. move the normaliser into setup as well.

```
setup() {
  const {$t} = usei18n();
  const subscribersNormalizer = data => {
 ....
}
```

**Review by Super-Chama — CHANGES_REQUESTED** [review]

**Review by Super-Chama — APPROVED** [review]

File: src/client/src/orangehrmPimPlugin/components/SaveEmployeeReportTo.vue [hunk 6]
```diff
@@ -20,9 +20,20 @@
...
 <template>
   <div class="orangehrm-horizontal-padding orangehrm-top-padding">
-    <oxd-text tag="h6" class="orangehrm-main-title"
-      >{{ $t('general.add') }} {{ type }}</oxd-text
+    <oxd-text
+      v-if="type === 'Supervisor'"
+      tag="h6"
+      class="orangehrm-main-title"
     >
+      {{ $t('general.add') }} {{ $t('pim.supervisor') }}
```
Comment (RajithaKumara) [inline src/client/src/orangehrmPimPlugin/components/SaveEmployeeReportTo.vue:28]: don't concatenate lang strings

File: src/client/src/orangehrmTimePlugin/components/TimesheetActions.vue [hunk 7]
```diff
@@ -76,9 +76,25 @@ export default {
...
+        let actionName = item.action?.label;
         const firstName = item.performedEmployee?.firstName;
         const lastName = item.performedEmployee?.lastName;
 
+        switch (actionName) {
+          case 'Submitted':
+            actionName = $t('time.submitted');
+            break;
+          case 'Rejected':
+            actionName = $t('leave.rejected');
```
Comment (RajithaKumara) [inline src/client/src/orangehrmTimePlugin/components/TimesheetActions.vue:88]: *

File: src/plugins/orangehrmDashboardPlugin/test/Dao/ChartDaoTest.php [hunk 8]
```diff
@@ -74,7 +74,7 @@ public function testGetEmployeeDistributionBySubunit(): void
         $result = $this->chartDao->getEmployeeDistributionBySubunit();
 
         $this->assertTrue(is_array($this->chartDao->getEmployeeDistributionBySubunit()));
-        $this->assertCount(11, $result);
+        $this->assertCount(9, $result);
         $this->assertEquals(3, $result[0]->getCount());
         $this->assertEquals('Engineering', $result[1]->getSubunit()->getName());
         $this->assertEquals(7, $result[2]->getSubunit()->getId());
```
Comment (RajithaKumara) [inline src/plugins/orangehrmDashboardPlugin/test/Dao/ChartDaoTest.php:80]: add test case for the bug fix

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**